pom.xml

```diff
@@ -15,21 +15,21 @@
 	<description>Demo project for Sentry in Java Spring Boot with Logback</description>
 
 	<properties>
-		<java.version>11</java.version>
+		<java.version>1.8</java.version>
```

Comment:
open for debate on what we should leave here as a default. I think we do Android demo's (1.8) before we run Java demo.

Reply:
for backend integrations, both would work, the majority of the Java apps are still 1.8 though

Reply:
I couldn't get the app to run if it says `11`. I remember with the android demo's I wrote, "Do not use JDK +11" or something like that.

As a team we might as well all be on the same version. We run the Android demo more often, and that's been pushing all new hires to go on 1.8. Vu and I are now on 1.8 .